nixos/modules/hardware/sensor/iio.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  ###### interface
+
+  options = {
+    hardware.sensor.iio = {
+      enable = mkEnableOption "Enable this option to support IIO sensors.";
```

Comment:
`mkEnableOption` expects the parameter to be the name of the module; it would generate a description of the form "Whether to enable Enable this option ...".